Podfile.lock

```diff
@@ -171,7 +171,7 @@ SPEC CHECKSUMS:
   NSObject+Rx: 2a9cd801d9c847e6d2486cbad8d7701b67834e70
   ORStackView: 5df6b1b990b0648d8ef6f69f89361ea8648e8f64
   Quick: 5d290df1c69d5ee2f0729956dcf0fd9a30447eaa
-  Reachability: 5b69def55ffa04d6f9425a6da1f47338b60557b2
+  Reachability: 2fc2badcac191cd1a15044bc7fe45aa96c499ece
```

Comment:
Considering tests pass, and there's not been a commit on the fork - I'm 👍 with this - but I'd recommend _you_ `rm -rf Pods` ash, http://artsy.github.io/blog/2016/05/03/podspec-checksums/